### PR TITLE
fix: replace require with readFileSync

### LINKED ISSUE #25: Doesn't work well with webpack
Because of dynamic require webpack can't work well with this package.
image
image

Can it be replaced with e.g. `fs.readFile`?

### PR DESCRIPTION
**Pull request by iljapostnovs**
- fix #25

### CODE REVIEW

**Comment by iljapostnovs** [comment]
Fix for #25

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "rc-config-loader",
-  "version": "4.1.2",
+  "version": "4.1.3",
```
Comment (azu) [inline package.json]:
```suggestion
  "version": "4.1.2",
```

No need to change.

**Comment by azu** [comment]
Thanks!

- 🎉 https://github.com/azu/rc-config-loader/releases/tag/v4.1.3

**Comment by iljapostnovs** [comment]
> Thanks!
> 
> * 🎉 https://github.com/azu/rc-config-loader/releases/tag/v4.1.3

Thanks to you too for the product and fast responses!
Keep up the good work! :)

**Pull request merged**